release-notes/v1.4.0.md

```diff
@@ -33,3 +33,4 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
 |CIDG-1305|Volume Mutator| Error validation in multiple edits | If multiple fields of PVC are edited, and if one of them is incorrect, partial edits are not done, and it results in a failure. `kubectl get events` again can be used for detecting such failure| Retry the operation of edit with all proper values by looking at the kubectl events logs|
 |CIDG-1302|Volume Mutator| Incorrect PVC edit| If there is a validation failure due to incorrect PVC edit, then the operation does not stop until the user corrects the edit operation. This is a limitation for now.|
 |CIDG-1327|Replication| System limitation|Handling system limitation of RCG| If more than 60 volumes are added into a RCG (Remote Copy Group) as part of parallel volume addition, due to the stop/start of RCG internally by CSP, the backend array can reach system limits and can cause PVC creation failure. Limit the number of PVCs to be added |
+|BZ-134504|csi.k8s| More online FC ports caused huge time for mount| Added a new option to read a list of FC ports from env variable "FC_PORTS_LIST" .| This is fixed with `quay.io/hpestorage/hpe3parprimera-csp:v1.2.1` and will be part of next release of csi-driver. `FC_PORTS_LIST` env variable needs to be added in `primera3par-csp` deployment.|
```

Comment:
Can you update the description with :

By default , the CSP creates vluns on all the FC ports that it sees. In some environments, with large number of fc ports this leads to increased mount time.  Added a new option to read a list of FC ports from env variable "FC_PORTS_LIST", to control what FC ports are considered for vlun creation

Reply:
Done.